scripts/integration/greptimedb/test.yaml

```diff
@@ -9,4 +9,12 @@ runner:
     GREPTIMEDB_HTTP: http://greptimedb:4000
 
 matrix:
-  version: ['latest']
```

Comment:
Non-blocking: Is there a `stable` tag?

Reply:
There isn't :/ (I also was wondering).

It's a fairly new project. I want to say we might have run into something like this with it before as well.